series/supportedseries.go

```diff
@@ -293,6 +294,33 @@ func VersionSeries(version string) (string, error) {
 	return "", errors.Trace(unknownVersionSeriesError(version))
 }
 
+// WindowsVersionSeries returns the series (eg: win2012r2) for the specified version
+// (eg: Windows Server 2012 R2 Standard)
+func WindowsVersionSeries(version string) (string, error) {
+	if version == "" {
+		return "", errors.Trace(unknownVersionSeriesError(""))
+	}
+	for _, val := range windowsVersionMatchOrder {
+		if strings.HasPrefix(version, val) {
+			return windowsVersions[val], nil
```

Comment:
I missed this earlier. It seems that the actual issue is that we *aren't* matching the full version string.
we're having to grab a prefix of the version string (which is not strictly unique).

Do we know if we're always matching the longest substring? Would it be possible to get rid of windowsVersionMatchOrder and just doi
matchedPrefix := ""
matchedSeries := ""
for prefix, series  := range windowsVersions {
  if len(prefix) > len(matchedPrefix) && strings.HasPrefix(version, prefix) {
    matchedPrefix = prefix
    matchedSeries = series
}
return series, nil

Thoughts?